### PR TITLE
pycon-ca-2012

### PR DESCRIPTION
**Pull request by logston**

### CODE REVIEW

File: data/pycon-ca-2012/category.json [hunk 1]
```diff
@@ -1,7 +1,5 @@
...
-  "url": "http://2012.pycon.ca/", 
-  "slug": "pycon-ca-2012", 
-  "start_date": "2012-11-09"
+  "slug": "pycon-ca-2012",
+  "title": "PyCon CA 2012",
+  "description": "PyCon Canada is an initiative to strengthen the Canadian Python\ncommunity.\n\nOur main goals:\n\n::\n\n    Support the efforts of Montreal Python as they prepare to host PyCon North America in 2014 and 2015. This is the first time that the main PyCon conference is going to be outside of the US. Let's take this opportunity to showcase the Canadian Python community, and step up to the challenge as speakers and volunteers.\n    Strengthen the Canadian Python community by providing more opportunities for us to share knowledge and ideas, encourage support and education for speaking at conferences, and increase the visibility of developers, organizations, and companies within the community.\n    Host an annual Canadian PyCon conference, starting now - 2012!\n\nPyCon Canada is entirely run by volunteers who are passionate about\nthese goals. We hope to see you at the 2012 conference!\n"
```
Comment (redapple) [inline data/pycon-ca-2012/category.json:4]: This description conversion did not turn out great:

```
PyCon Canada is an initiative to strengthen the Canadian Python
community.

Our main goals:

::

    Support the efforts of Montreal Python as they prepare to host PyCon North America in 2014 and 2015. This is the first time that the main PyCon conference is going to be outside of the US. Let's take this opportunity to showcase the Canadian Python community, and step up to the challenge as speakers and volunteers.
    Strengthen the Canadian Python community by providing more opportunities for us to share knowledge and ideas, encourage support and education for speaking at conferences, and increase the visibility of developers, organizations, and companies within the community.
    Host an annual Canadian PyCon conference, starting now - 2012!

PyCon Canada is entirely run by volunteers who are passionate about
these goals. We hope to see you at the 2012 conference!

```

File: data/pycon-ca-2012/videos/dancing-with-big-data-disco-inferno.json [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+      "length": 0
     }
-  ], 
-  "source_url": "https://www.youtube.com/watch?v=D3VFq97NhEk", 
-  "tags": [], 
+  ],
+  "source_url": "https://www.youtube.com/watch?v=D3VFq97NhEk",
+  "tags": [],
   "speakers": [
     "Tim Spurway &  Mazdak Rezvani"
```
Comment (redapple) [inline data/pycon-ca-2012/videos/dancing-with-big-data-disco-inferno.json:28]: ideally, could be split into 2 speakers

File: data/pycon-ca-2012/videos/how-you-can-contribute-to-python.json [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
-  "quality_notes": "", 
-  "language": "English", 
-  "copyright_text": "", 
-  "thumbnail_url": "http://i2.ytimg.com/vi/i-dZ2pxJNFM/hqdefault.jpg", 
-  "duration": null, 
+  "id": 1578,
+  "category": "PyCon CA 2012",
+  "slug": "how-you-can-contribute-to-python",
+  "title": "How you can contribute to Python",
+  "summary": "Did you know that you could contribute to Python? From documentation\n\nwork to patch reviews to new features to blogging, there are a number of\n\nways you can give back to Python. This talk for beginners will walk you\n\nthrough these ways with concrete demonstrations.\n",
```
Comment (redapple) [inline data/pycon-ca-2012/videos/how-you-can-contribute-to-python.json:6]: ideally, extra newlines could be removed (they are in the original Markdown too)

File: data/pycon-ca-2012/videos/wilderness-labs.json [hunk 4]
```diff
@@ -1,31 +1,31 @@
...
-  "quality_notes": "", 
-  "language": "English", 
-  "copyright_text": "", 
-  "thumbnail_url": "http://i3.ytimg.com/vi/fkQgChJY7Xk/hqdefault.jpg", 
-  "duration": null, 
+  "id": 1603,
+  "category": "PyCon CA 2012",
+  "slug": "wilderness-labs",
+  "title": "Wilderness Labs",
+  "summary": "Lessons learned and inspiration from developing Django applications\n\nquickly in isolated locations via Wilderness Labs and other zero\n\nbudget projects.\n",
```
Comment (redapple) [inline data/pycon-ca-2012/videos/wilderness-labs.json:6]: extra newlines could be removed.

**Comment by redapple** [comment]
Only the description of the conference (`data/pycon-ca-2012/category.json`) needs editing.
The rest looks ok to me.

**Comment by codersquid** [comment]
Thanks @redapple, I've created #109 for cleaning the data.

**Pull request merged**